beta-src/src/utils/draw-arrow.ts

```diff
@@ -0,0 +1,46 @@
+export default function drawArrow(actionType, source, svgElement, target) {
```

Comment:
there is not typing going on here. we need to define an interface, otherwise typescript is irrelevant.